images/build/Dockerfile

```diff
@@ -133,7 +134,7 @@ RUN set -ex \
  && ln -s 3.0 3 \
  && ln -s $sdksDir/$DOT_NET_CORE_30_SDK_VERSION $NET_CORE_APP_30/sdk \
  # LTS sdk <-- LTS runtime's sdk
- && ln -s 2.1 lts \
+ && ln -s 2.2 lts \
```

Comment:
2.1 is the actually LTS - https://dotnet.microsoft.com/download/dotnet-core

Reply:
thanks .. I am reverting this change